src/apps/search-result/search-result.test.ts

```diff
@@ -0,0 +1,127 @@
+const coverUrlPattern = /^https:\/\/res\.cloudinary\.com\/.*\.(jpg|jpeg|png)$/;
+
+describe("Search Result", () => {
+  it("Should render the search results as expected", () => {
+    cy.visit(
+      "/iframe.html?id=apps-search-result--search-result&args=pageSizeDesktop:2;pageSizeMobile:2"
+    );
+
+    cy.log("Check length of search result list:");
+    cy.get(".search-result-page__list").find("li").should("have.length", 2);
+
+    cy.log("Do the search results have images?");
+    cy.get(".search-result-page__list .search-result-item img")
+      .should("have.attr", "src")
+      .and("match", coverUrlPattern);
+
+    cy.log("Does the search result have favourite buttons?");
```

Comment:
I like this way of attaching something humanly readable to an assertion. It reminds me a bit of PHPUnit 😄 . We could also do this with code comments but this puts it in the Cypress UI which may make it easier to work with in GitHub Actions in the like.

Reply:
👍